MdeModulePkg/Library/DeviceManagerUiLib/DeviceManager.c

```diff
@@ -619,7 +619,8 @@ CreateDeviceManagerForm(
         TokenHelp = HiiSetString (HiiHandle, 0, String, NULL);
         FreePool (String);
 
-        FormSetGuid = ((EFI_IFR_FORM_SET *)Ptr)->Guid;

```

Comment:
@changab Do you know why this hasn't been discovered before?
Is it usually possible to assign Guids with `=`?
Or does it only fail for me because of our RISC-V toolchain?

Reply:
I need to verify that this is still an error with the newest toolchain.

Reply:
Yeah, still an error.